src/dbSta/src/dbNetwork.cc

```diff
@@ -1351,7 +1351,52 @@ NetTermIterator* dbNetwork::termIterator(const Net* net) const
   return new DbNetTermIterator(net, this);
 }
 
+class AccumulatePins : public PinVisitor
+{
+ public:
+  AccumulatePins(const sta::Network* network) { network_ = network; }
+  virtual void operator()(const sta::Pin* pin)
+  {
+    accumulated_pins_.insert(pin);
+  }
+  std::set<const sta::Pin*> accumulated_pins_;
+  const sta::Network* network_;
+};
+
+/* find equivalent modnet for a dbnet */
+dbModNet* dbNetwork::getDbModNetFromDbNet(dbNet* db_net)
```

Comment:
Many mod nets may be part of one flat dbNet.  The appears to just pick an arbitrary one.  What is the intention for how this will be used?

Reply:
A dbInstance may be connected to none, many or just one dbModNet. All the dbModNets can be reached from each other. So we just need to find one (anyone). Given any modnet we can get from it all the pins in its hierarchy (which is what the accumulate pins functor is doing, notice how that functor traverses through the hierarchy).  The intent is (1) Get a mod net for a dbNet (again any modnet). (2) Provide support for getting all the pins related to a modnet (ie the whole set of pins in the hierarchy). This code is not used in this application so can be removed from this pull request.

Reply:
removed